pom.xml

```diff
@@ -0,0 +1,548 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Ingest API</name>
+    <inceptionYear>2019</inceptionYear>
+    <organization>
+        <name>Connexta</name>
+    </organization>
+    <licenses>
+        <license>
+            <name>GNU Lesser General Public v3</name>
+            <url>http://www.gnu.org/licenses/lgpl.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <groupId>com.connexta.ingest</groupId>
+    <artifactId>ingest-api</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
```

Comment:
❗️ First version should be `1.0.1-SNAPSHOT`

Reply:
I disagree. Since this API is still in development it should start at 0.1.0 according to the [semver website](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase). If we decide to change any of the min / max length for the title / bytes, then that'd be a breaking change forcing our second version to be 2.0.0.

Reply:
only if you release it. As long as you don't release it, you will still be on 1.0.1-SNAPSHOT working towards 1.0.1.